artiq/coredevice/mirny.py

```diff
@@ -140,3 +140,144 @@ def write_ext(self, addr, length, data):
         if length < 32:
             data <<= 32 - length
         self.bus.write(data)
+
+
+class Almazny:
+    """
+    Almazny (High frequency mezzanine board for Mirny)
+
+    :param mirny - Mirny device Almazny is connected to
+    """
+
+    # pin map
+    SER_MOSI = 0
+    SER_CLK = 1
+    REG_LATCH_BASE = 2
+    REG_CLEAR = 6
+
+    mezzio_reg = 0x03  # register addr for Mezz_IO 0~7
```

Comment:
I'm not sure if we're going to support those class attributes in NAC3 (@pca006132 ?).
If not it may be better to put them in the module with a ``ALMAZNY_*`` prefix to reduce divergence.

Reply:
> I'm not sure if we're going to support those class attributes in NAC3, which should also be kernel invariants in this case (@pca006132 ?).

We can, but probably not recently as there are other higher priority tasks.